interface-definitions/dns-forwarding.xml.in

```diff
@@ -598,6 +598,19 @@
                 </properties>
                 <defaultValue>3600</defaultValue>
               </leafNode>
+              <leafNode name="network-timeout">
+                <properties>
+                  <help>Number of milliseconds to wait for a remote authoritative server to respond</help>
+                  <valueHelp>
+                    <format>u32:0-4294967295</format>
+                    <description>Network timeout in milliseconds</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 0-4294967295"/>
```

Comment:
Does 0-4294967295 milliseconds really make sense?

I'd set something like 10-10000 (10ms - 10s) given that we have a perfectly compatible default value of 1,5 seconds this would make more sense to me.

Reply:
I am also fine with this, I only had an issue with the broad range of uint32 ;).